pkg/kubelet/kuberuntime/kuberuntime_gc_test.go

```diff
@@ -34,6 +34,19 @@ func TestSandboxGC(t *testing.T) {
 	fakeRuntime, _, m, err := createTestRuntimeManager()
 	assert.NoError(t, err)
 
+	fakePodGetter := m.containerGC.podGetter.(*fakePodGetter)
+	makeGCSandbox := func(pod *v1.Pod, attempt uint32, state runtimeapi.PodSandboxState) sandboxTemplate {
+		if pod.Name != "deleted" {
```

Comment:
Why not introduce an arguement to indicate whether to add the pod into pod getter?